src/schema/organization/types.js

```diff
@@ -30,6 +30,7 @@ const OrganizationType = new GraphQLObjectType({
     members,
     repositories,
     contributors,
+    contributor,
```

Comment:
why `contributor` is here?

Reply:
Because was the best place that I found, based on information needed

Reply:
It's confuse, but i understand, i thinking that is better make the query for org id, but in this moment, that is enough